iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/visitors/CommentLocationVisitor.java

```diff
@@ -90,9 +91,22 @@ public void visitComment(InputFileContext ctx, YamlTree tree) {
 
   private void processComment(InputFileContext ctx, Comment comment) {
     if (IS_LINE_NUMBER.matcher(comment.contentText()).matches()) {
+      // there is only line comment
       int lineCommentLocation = comment.textRange().start().line();
       var lineCommentValue = Integer.parseInt(comment.contentText());
       shifter.addShiftedLine(ctx, lineCommentLocation, lineCommentValue);
+    } else {
+      var matcher = CONTAINS_LINE_NUMBER.matcher(comment.contentText());
+      if (matcher.find()) {
```

Comment:
Nice approach how you match it (using combination of `.find()` and `$`.)

Reply:
Thanks for this comment. It reminds me about one another edge case